src/data/collection.json

```diff
@@ -2719,5 +2719,27 @@
 		"author": "Cider",
 		"notes": "Deliberately vulnerable CI/CD environment. Hack CI/CD pipelines, capture the flags.",
 		"badge": "cider-security-research/cicd-goat"
+	},
+	{
+		"url": "http://ffuf.me/",
+		"name": "FFUF.me",
+		"collection": [
+			"online",
+			"offline",
+			"container"
+		],
+		"technology": [
+			"PHP",
+			"Docker"
+		],
+		"references": [
+			{
+				"name": "source",
```

Comment:
One of:
https://github.com/OWASP/OWASP-VWAD/blob/f55137da6cb537a071e8e208c0616ef48b51105c/schema.json#L104-L116

Reply:
Ok, I should have checked teh schema.